homework/Vasili_Zasinets/src/test/java/com/tutrit/java/quickstart/homework/arrayservice/ArrayServiceTest.java

```diff
@@ -0,0 +1,45 @@
+package com.tutrit.java.quickstart.homework.arrayservice;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ArrayServiceTest {
+    public int[] array;
+    Logger log = LoggerFactory.getLogger("ArrayServiceTest");
+    ArrayService arrayService = new ArrayService();
+
+    @Before
+    public void setUp() {
+        this.array = arrayService.create(10);
+    }
+
+    @Test
+    public void fillRandomly() {
+        arrayService.fillRandomly(array);
+        for (int j : array) {
+            log.info("array i {} ", j);
+        }
+    }
+
+    @Test
+    public void printArray() {
```

Comment:
you could check if array created is not empty.

Reply:
fixed